### ISSUE TITLE
URL for chapter viewing should be based on chapter number instead of id

### ISSUE THREAD

**Issue by robink** [spreadthesource/wooki#84]
URL should look like :
chapter/bookid/1
chapter/bookid/2
chapter/bookid/3
and so on...

instead of
chapter/bookid/13
chapter/bookid/19
chapter/bookid/240
chapter/bookid/492

**Comment by ccordenier** [comment]
I Totally agree with that, but this will require session information even for readers.

**Comment by ccordenier** [comment]
Postponed to 1.0 for priority reasons
